selenium-examples/pom.xml

```diff
@@ -22,6 +22,18 @@
             <version>5.7.0-M1</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
@coleve27 not sure what this `sauce_testng` dependency is but I don't believe that we need it.